meta-resin-common/recipes-support/resin-device-uuid/resin-device-uuid/resin-device-uuid

```diff
@@ -41,7 +41,7 @@ if [ -z "$CONFIG_PATH" ]; then
     exit 1
 fi
 
-if [ ! -f "$CONFIG_PATH" ]; then
+if [ ! -f "$CONFIG_PATH" ] || [ ! -s "$CONFIG_PATH" ]; then
```

Comment:
Because in our normal images there can be the case that the path /mnt/boot/config.json does not exist, so we have to check for that too.